content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -816,6 +816,48 @@ You can optionally define a [custom unique constraint name](/concepts/components
 
 </Admonition>
 
+#### Composite type unique constraints
+
+When using the MongoDB provider in version `3.12.0` and later, you can define a unique constraint on a field `field` of a [composite type](#defining-composite-types) `compositeType` using the syntax `@@unique([compositeType.field])`. As with other fields, composite type fields can be used as part of a multi-column unique constraint.
```

Comment:
This is a pretty weird example (calling the field `field`, and the type `compositeType`), although of course correct.

Reply:
It's not really an example, more a demonstration of the syntax. What would you suggest?

Reply:
Thought I'd updated this, must have missed one.